kolibri/core/exams/test/test_exam_api.py

```diff
@@ -233,6 +233,20 @@ def test_exam_with_invalid_exercise_id(self):
         exam["question_sources"].append(
             {
                 "exercise_id": "e1",
+                "question_id": "e1e1e1e1e1e1e1e1e1e1e1e1e1e1e1e1",
```

Comment:
For the valid UUIDs in the fixtures, can you use a realistic UUID? Either create one via the `uuid` module or generate and copy-paste it into the code.